src/plugins/content_management/demo/todo/todos_client.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { v4 as uuidv4 } from 'uuid';
+import type { CrudClient } from '../../public/crud_client';
+import type { CreateIn, DeleteIn, GetIn, SearchIn, SearchOut, UpdateIn } from '../../common';
+
+export interface Todo {
+  id: string;
+  title: string;
+  completed: boolean;
+}
+
+export class TodosClient implements CrudClient {
```

Comment:
For these stories I am creating a client side `TodosClient` to be used as a data source instead of the server-side registry. 

Current types approach didn't work well the TodosClient implement is mostly untyped now